### DISCUSSION TITLE
json

### DISCUSSION THREAD

**Discussion by YangSuBin1101** [coinfoundry/miningcore#1135] [General]
![캡처](https://user-images.githubusercontent.com/77947169/150491191-61d9d076-1eb2-423f-85f3-1356aa0445a8.PNG)

miner display only new job.
why don't display accept?

pool difficulty is 0.01
min diff in vardiff is not exist
max diff in vardiff is null

NOW, display ERROR Message
1. JSON decode failed(miner)
2. refict low difficutity(miner and pool)
3. duplicate share(miner and pool)
4. reset perfomance stats for miner(pool)
5. Connection json error state: Unexpected character encountered while parsing value: . Path '', line 0, position 0. 
Banning client for sending junk (pool)

Is it related to these new messages? also, tell me Any solution to the error message you know.
thank you

**Comment by oliverw** [comment]
Connection json error state is usually script kiddies trying to hack your server. Happens all the time. Nothing to worry about.
